meta-lmp-base/recipes-core/base-files/base-files/tmpfiles.conf

```diff
@@ -0,0 +1,6 @@
+d  /var/backups  755 root root -
+d  /var/lib/misc 755 root root -
+d  /var/local    755 root root -
+d  /var/spool    755 root root -
+L+ /var/run      -   -    -    - ../run
+L+ /var/lock     -   -    -    - ../run/lock
```

Comment:
These 2 are already managed by systemd, and since tmpfiles.conf is only relevant with systemd, we can probably drop it (to avoid having systemd to recreate the links):


```
root@intel-corei7-64-unknown:/usr# grep -R "var/run" lib/tmpfiles.d/
lib/tmpfiles.d/var.conf:L /var/run - - - - ../run
root@intel-corei7-64-unknown:/usr# grep -R "var/lock" lib/tmpfiles.d/
lib/tmpfiles.d/legacy.conf:L /var/lock - - - - ../run/lock
```

Reply:
Fixed